src/Tickets/Seating/Orders/Seats_Report.php

```diff
@@ -233,23 +237,20 @@ public function add_seats_row_action( $actions, $post ): array {
 	 * @return bool Whether the upsell should show or not.
 	 */
 	protected function should_show_upsell(): bool {
-		$seating = get_resource( 'tec-seating' );
+		$service_status = tribe( Service::class )->get_status( true );
```

Comment:
Why force a refresh here? The status is memoized and it will not change in the request.